recipes/install.rb

```diff
@@ -89,7 +89,7 @@
 # The following dependencies are required to run spark-sql with parquet and orc. We install them here so that users don't have to do it from their notebooks/jobs
 # https://mvnrepository.com/artifact/org.spark-project.hive/hive-exec/1.2.1.spark2
 # http://central.maven.org/maven2/org/iq80/snappy/snappy/0.4/
-files= "parquet-encoding-#{node['hadoop_spark']['parquet_version']}.jar, parquet-common-#{node['hadoop_spark']['parquet_version']}.jar, parquet-hadoop-#{node['hadoop_spark']['parquet_version']}.jar, parquet-jackson-#{node['hadoop_spark']['parquet_version']}.jar, parquet-column-#{node['hadoop_spark']['parquet_version']}.jar, parquet-format-#{node['hadoop_spark']['parquet_format_version']}.jar, spark-hive_#{node['scala']['version']}-#{node['hadoop_spark']['version']}.jar,snappy-0.4.jar, spark-avro_#{node['hadoop_spark']['spark_avro_version']}.jar, spark-tensorflow-connector_#{node['hadoop_spark']['tf_spark_connector_version']}.jar, hudi-client-#{node['hive2']['hudi_version']}.jar, hudi-hive-bundle-#{node['hive2']['hudi_version']}.jar, hudi-utilities-#{node['hive2']['hudi_version']}.jar, hudi-common-#{node['hive2']['hudi_version']}.jar, hudi-spark-#{node['hive2']['hudi_version']}.jar, spark-avro_#{node['hadoop_spark']['databricks_spark_avro_version']}.jar, delta-core_#{node['hadoop_spark']['databricks_delta_version']}.jar"
+files= "parquet-encoding-#{node['hadoop_spark']['parquet_version']}.jar, parquet-common-#{node['hadoop_spark']['parquet_version']}.jar, parquet-hadoop-#{node['hadoop_spark']['parquet_version']}.jar, parquet-jackson-#{node['hadoop_spark']['parquet_version']}.jar, parquet-column-#{node['hadoop_spark']['parquet_version']}.jar, parquet-format-#{node['hadoop_spark']['parquet_format_version']}.jar, spark-hive_#{node['scala']['version']}-#{node['hadoop_spark']['version']}.jar,snappy-0.4.jar, spark-avro_#{node['hadoop_spark']['spark_avro_version']}.jar, spark-tensorflow-connector_#{node['hadoop_spark']['tf_spark_connector_version']}.jar, hudi-spark-bundle-#{node['hive2']['hudi_version']}.jar, spark-avro_#{node['hadoop_spark']['databricks_spark_avro_version']}.jar, delta-core_#{node['hadoop_spark']['databricks_delta_version']}.jar"
```

Comment:
This way of downloading jar files won't work during updates. If we don't bump the Spark version we will have multiple Jars of Hudi in `/srv/hops/spark/jars`. 
Maybe we can have a `hopsworks_jars` dir which we recreate every time we run the recipe and then link in `/srv/hops/spark/jars`. `hopsworks_jar` will contain only the jars above.

Reply:
As this is the same issue for all jar files we deploy, I suggest that we fix that in a separate commit.

Reply:
I added some code for deleting the old jars before downloading. I believe that should fix it for the upgrade. I'll run a VM creation test on this.